src/main/java/rx/Single.java

```diff
@@ -605,6 +606,42 @@ public void call(SingleSubscriber<? super T> te) {
     }
 
     /**
+     * Returns a {@link Single} that invokes passed function and emits its result for each new Observer that subscribes.
+     * <p>
+     * Allows you to defer execution of passed function until Observer subscribes to the {@link Single}.
+     * It makes passed function "lazy".
+     * Result of the function invocation will be emitted by the {@link Single}.
+     * <dl>
+     *   <dt><b>Scheduler:</b></dt>
+     *   <dd>{@code fromCallable} does not operate by default on a particular {@link Scheduler}.</dd>
+     * </dl>
+     *
+     * @param func
+     *         function which execution should be deferred, it will be invoked when Observer will subscribe to the {@link Single}.
+     * @param <T>
+     *         the type of the item emitted by the {@link Single}.
+     * @return a {@link Single} whose {@link Observer}s' subscriptions trigger an invocation of the given function.
+     */
+    public static <T> Single<T> fromCallable(final Callable<? extends T> func) {
```

Comment:
This should start out as experimental unless the RxJava contributors want to fast-track this.